modules/reactor/test/tests/meshgenerators/reporting_id/hexagonal_id/tests

```diff
@@ -0,0 +1,42 @@
+[Tests]
+  design = 'meshgenerators/HexIDPatternedMeshGenerator.md'
+  issues = '#19217'
+  [assembly.cell]
```

Comment:
same question about recover

Reply:
@loganharbour 
What is the purpose of `recover` here?
This tests file was made based on the existing tests files in the reactor module, which set `recover` as `false`.

Reply:
`recover` is a class of tests that run for half their timesteps, create a checkpoint, stop then recover from the checkpoint. 

Those tests were either steady state tests (which dont recover) or just mesh generation tests (also no recover)